srcpkgs/python3-signedjson/template

```diff
@@ -1,15 +1,20 @@
 # Template file for 'python3-signedjson'
 pkgname=python3-signedjson
-version=1.1.0
-revision=4
-wrksrc="python-signedjson-${version}"
+version=1.1.1
+revision=1
+wrksrc="signedjson-${version}"
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm"
 depends="python3-canonicaljson>=1.0.0 python3-unpaddedbase64>=1.0.1
- python3-pynacl"
+ python3-pynacl python3-typing_extensions"
+checkdepends="${depends} python3-importlib_metadata python3-pytest"
```

Comment:
Why is this only in checkdepends when it's imported in __init__.py? Anyway, since you are patching importlib_metadata, we shouldn't need to this dependency. I think we should import this patch instead of running sed in post_patch: https://github.com/matrix-org/python-signedjson/commit/c40c83f844fee3c1c7b0c5d1508f87052334b4e5

Reply:
Good catch, thanks.